testing/test.js

```diff
@@ -1,6 +1,6 @@
 const test = require('tape');
 
 test('Tape is working', (t) => {
-  t.pass();
+  t.pass('Tape is working');
```

Comment:
No need to repeat 'Tape is working' here. Ideally you would add a small equals test like comparing 1 to 1 just to make sure everything is working properly.

Reply:
@MynahMarie resolved both issues. Deleted that second test file as it was empty, renamed dummy to filler cook data.